dbfit-java/informix/src/main/java/dbfit/environment/InformixFunctionStatement.java

```diff
@@ -2,19 +2,19 @@
 
 import java.sql.*;
 
-import dbfit.fixture.StatementExecution;
+import dbfit.util.PreparedDbStatement;
 
-public class InformixFunctionStatementExecution extends StatementExecution {
+public class InformixFunctionStatement extends PreparedDbStatement {
```

Comment:
Do we have any standard for naming subtypes? Should we name this `InformixFunctionDbStatement`?